crates/core/Cargo.toml

```diff
@@ -37,6 +43,7 @@ browser = [
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+serde_urlencoded = "0.7.1"
```

Comment:
This can be a optional dependency that's enabled with the `liveview-channels` feature.